package.json

```diff
@@ -24,8 +24,10 @@
     "test:compatibility:local:test": "mkdir -p build/reports/compatibility && cross-env TESTS=localCompatibility cucumber-js src/test/compatibility/features/ --require 'src/test/common/steps/*.js' --require 'src/test/compatibility/config' -f json:build/reports/compatibility-report.json",
     "test:compatibility:local:report": "node src/test/compatibility/local-compatibility-report.js",
     "test:accessibility": "node 'src/test/a11y/test-suite.js'",
-    "test:browser": "npm run test:wait && npm-run-all -s test:smoke test:accessibility test:compatibility:local test:acceptance test:acceptance:report",
+    "test:browser": "npm run test:wait && npm run test:startWiremock && npm-run-all -s test:smoke test:accessibility test:compatibility:local test:acceptance test:acceptance:report test:stopWiremock",
```

Comment:
Could maybe use `npm-run-all` for the first two commands as well here:
```
"test:browser": "npm-run-all -s test:wait test:startWiremock && npm-run-all -s test:smoke test:accessibility test:compatibility:local test:acceptance test:acceptance:report test:stopWiremock"
```

Reply:
Updated to use npm run all.